platform/ios/docs/guides/UserInteractions.md

```diff
@@ -0,0 +1,47 @@
+# User Interactions in the Mapbox iOS SDK
```

Comment:
Titles of guides don’t need to say “Mapbox iOS SDK”, since the guides are part of the iOS SDK docset.